python/tk_multi_screeningroom/screeningroom.py

```diff
@@ -53,20 +56,32 @@ def _launch_rv(base_url, cmd, source=None, path_to_rv=None):
     if source:
         args.append(source)
 
+    combined_args = " " + " ".join(args)
     # We should encode the args so that no special characters exist in the command
-    encoded_args = (" " + " ".join(args)).encode("hex")
+    # Encode the string into binary so that we can convert the args to a hex
+    binary_str = six.ensure_binary(combined_args)
+    # Encode the binary into a hex
+    encoded_args = codecs.encode(binary_str, "hex")
+    # Now convert the binary back into a string which contains the hex representing the args.
+    hex_encoded_args_string = six.ensure_str(encoded_args)
```

Comment:
As Python3 now handles binary and string separately, I now need to convert the string args over to binary, hex encode them, and then convert them back to string.